tests/keras/utils/data_utils_test.py

```diff
@@ -0,0 +1,24 @@
+import os
+import pytest
+from keras.utils.data_utils import get_file
+from keras.utils.data_utils import hash_file
+from keras.utils.data_utils import validate_file
+from keras import activations
+from keras import regularizers
+
+
+def test_file_io():
+    dirname = 'cifar-10-batches-py'
+    origin = 'http://www.cs.toronto.edu/~kriz/cifar-10-python.tar.gz'
```

Comment:
I wasn't sure how to do that, should I create a local file, then load it with a file url? What location would be safe?